### PR TITLE
[FIX] hr_timesheet,repair: fix field width issue

### PR DESCRIPTION
**Pull request by kcv-odoo**
Before this PR field width was not as expected because
its label defined in container(Ex: div,span)

This PR fix field width issue by giving o_td_label class
to container of label to properly compute its width

task-3032518

### CODE REVIEW

File: addons/hr_timesheet/__manifest__.py [hunk 1]
```diff
@@ -50,6 +50,9 @@
         'web.assets_backend': [
             'hr_timesheet/static/src/**/*',
         ],
+        'web.assets_frontend': [
```
Comment (IT-Ideas) [inline addons/hr_timesheet/__manifest__.py:53]: Shouldn't it move into `project.webclient` ?

**Review by IT-Ideas — COMMENTED** [review]
Thanks 🙂

File: addons/hr_timesheet/__manifest__.py (hunk 1 above)
Comment (kcv-odoo) [reply addons/hr_timesheet/__manifest__.py]: Done! Thanks :)

**Comment by IT-Ideas** [comment]
Thanks 🙂 
@robodoo r+

**Pull request closed without merging**